test/integration/scheduler/framework_test.go

```diff
@@ -126,11 +126,11 @@ var _ = framework.PostbindPlugin(&PostbindPlugin{})
 var _ = framework.UnreservePlugin(&UnreservePlugin{})
 var _ = framework.PermitPlugin(&PermitPlugin{})
 
-var scPlugin = &ScorePlugin{}
-
-// NewScorePlugin is the factory for score plugin.
-func NewScorePlugin(_ *runtime.Unknown, _ framework.FrameworkHandle) (framework.Plugin, error) {
-	return scPlugin, nil
+// NewScorePlugin returns a factory for score plugin with specified ScorePlugin.
+func NewScorePlugin(scorePlugin *ScorePlugin) framework.PluginFactory {
```

Comment:
do we really need to export those New* functions? should be enough if we do newScorePlugin etc.

Reply:
Agree :) Addressed it.